apps/meteor/ee/app/livechat-enterprise/server/hooks/onTransferFailure.ts

```diff
@@ -26,13 +28,21 @@ const onTransferFailure = async ({
 	}
 
 	cbLogger.debug(`Fallback department ${department.fallbackForwardDepartment} found for department ${department._id}. Redirecting`);
+	// TODO: find enabled not archived here
```

Comment:
Good catch! Should we plan to do this here or in a seperate PR?

Reply:
Yep!